core/services/job/runner_integration_test.go

```diff
@@ -1124,3 +1110,5 @@ func TestRunner_Error_Callback_AsyncJob(t *testing.T) {
 		require.Eventually(t, func() bool { return eiNotifiedOfDelete }, 5*time.Second, 10*time.Millisecond, "expected external initiator to be notified of deleted job")
 	}
 }
+
+func ptr[T any](t T) *T { return &t }
```

Comment:
Hoping we can drop these in two release :crossed_fingers: 
https://github.com/golang/go/issues/45624